composer.json

```diff
@@ -5,6 +5,9 @@
     "type"       : "wordpress-plugin",
     "license"    : "GPL-2.0+",
     "require"    : {
+        "wp-coding-standards/wpcs": "^1.0.0",
```

Comment:
These should be `require-dev` dependencies.

Reply:
fixed with 268daa4.